### DISCUSSION TITLE
Minor or patch release

### DISCUSSION THREAD

**Discussion by lucia3e8** [git-lfs/git-lfs#5610] [General]
Hi,
Could we have another minor release? It's been about 6 months since the last one and I'd really like a release that includes this PR https://github.com/git-lfs/git-lfs/pull/5357.

If we'd rather not cut a new minor release, what could I do to get that PR into the next patch release?

**Comment by chrisd8088** [comment]
Hey, thanks for the question and prompt.  It would be good for us to do a minor release in the not-too-distant future.

I happen to know the core LFS team will be busy through January, and that we also have to rework our Windows binary signing mechanism before we can do another release, but perhaps we can aim for something in February.
